process/mock/blockSizeComputationStub.go

```diff
@@ -0,0 +1,37 @@
+package mock
+
+type BlockSizeComputationStub struct {
```

Comment:
"// BlockSizeComputationStub -- " - add comments like this to avoid golangci warnings

Reply:
Done